src/main.js

```diff
@@ -340,4 +354,28 @@ function init() {
     renderOverlay();
 }
 
+function getMaxPage() {
+    let pageCalc = Math.ceil(nonFollowersList.length / UNFOLLOWERS_PER_PAGE);
+    return pageCalc < 1 ? 1 : pageCalc;
+}
+
+function nextPage() {
+    if (currentPage < getMaxPage()) {
+        currentPage++;
+        renderResults();
+    }
+}
+
+function getCurrentPageUnfollowers() {
+    let clonedList = [...nonFollowersList].sort((a, b) => (a.username > b.username ? 1 : -1));
```

Comment:
> Better to always use `const` instead of `let` unless the variable gets re-assigned :)

Reply:
Also, while it's true that this list is cloned, it's also being sorted.
So I think that should take precedence in the naming.
i.e. - `sortedList` would be better IMO.